postgresStore.go

```diff
@@ -0,0 +1,257 @@
+package timetracker
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type PostgresStore struct {
+	Db *sql.DB
+}
+
+func NewPostgresStore(conn string) (*PostgresStore, error) {
```

Comment:
Wouldn't it make sense to return a `*TaskStore` here? After all, if a `*PostgresStore` is a `*TaskStore`, that should be fine.